Update API of Docs in module, Time (#22581)
ballerina-platform/ballerina-lang: stdlib/time/src/main/ballerina/src/time/Module.md

This module provides implementations related to time, date, time zones, and durations. 

> The module has two main types as [Time](records/Time.html) and [TimeZone](records/TimeZone.html). The type `Time` represents a time associated with a given time zone. It has `time` and `zone` as attributes. The type `TimeZone` represents the time zone associated with a given time. It has `id` and `offset` as attributes. An `id` can be one of the following:

Reviewer: ```suggestion
The module has two main types as [Time](records/Time.html) and [TimeZone](records/TimeZone.html). The `Time` type represents a time associated with a given time zone. It has `time` and `zone` as its attributes. The `TimeZone` type represents the time zone associated with a given time. It has `id` and `offset` as its attributes. An `id` can be one of the following:
```

> * If `id` starts with '+' or '-', the ID is parsed as an offset. Offset can be specified in one of the following ways. +h, +hh, +hh:mm, -hh:mm, +hhmm, -hhmm, +hh:mm:ss, -hh:mm:ss, +hhmmss, -hhmmss

Reviewer: ```suggestion
* If `id` starts with '+' or '-', the ID is parsed as an offset. The offset can be specified in one of the following ways: +h, +hh, +hh:mm, -hh:mm, +hhmm, -hhmm, +hh:mm:ss, -hh:mm:ss, +hhmmss, -hhmmss
```

> * Also `id` can be a region-based zone ID. The format is '{area}/{city}' eg: "America/Panama". The zones are based on IANA Time Zone Database (TZDB) supplied data.

Reviewer: ```suggestion
* Also, `id` can be a region-based zone ID. The format is '{area}/{city}' e.g., "America/Panama". The zones are based on IANA Time Zone Database (TZDB) supplied data.
```
